packages/app-desktop/gui/NoteList/utils/UseAutoScroll.test.ts

```
@@ -0,0 +1,106 @@
+import useAutoScroll from './useAutoScroll';
+import { renderHook } from '@testing-library/react';
+
+type Props = {
+	selectedNoteId: string;
+	selectedFolderId: string;
+	targetIndex: number;
+	makeItemIndexVisible: (index: number)=> void;
```

ESLint

[error] 'index' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)